afdatetime/timezones.py

```diff
@@ -35,3 +35,38 @@ def find(self, lat, lng, dt):
         hours_str = ('-' if hours<0 else '+') + str(abs(hours)).rjust(2,'0')
         minutes_str = str(minutes).rjust(2,'0')
         return td, hours_str + ':' + minutes_str
+
+class DstAccurateTimeZone(object):
+    """
+        Finds the three character abbreviation of a timezone
+        given a lat, lng and datetime.
+    """
+
+    def __init__(self):
+        self._timezone_finder = TimezoneFinder()
+
+    def find(self, lat, lng, dt):
+        """Returns the abbreviated tz name
+
+        Args:
+         - lat -- latitude of location
+         - lng -- longitude of location
+         - dt -- datetime (as UTC offset varies with date)
+
+        Returns a dict containing the tz string, daylight savings time appropriate abbreviation, and 
+        a boolean whether the location is in DST.
+        """
+        tz_str = self._timezone_finder.timezone_at(lng=lng, lat=lat)
+        if not tz_str:
+            return None, None
```

Comment:
Should this not return either a single `None` or a dict with `None` values (like  
```
{'name': None, 'abbreviation': None, 'dst': None}
```